src/Uno.Templates/content/unoapp/Directory.Build.props

```diff
@@ -32,6 +32,9 @@
 		<IsMac>false</IsMac>
 		<IsMacCatalyst>false</IsMacCatalyst>
 		<IsWinAppSdk>false</IsWinAppSdk>
+
+		<!-- Improve build performance - https://devblogs.microsoft.com/visualstudio/vs-toolbox-accelerate-your-builds-of-sdk-style-net-projects/ -->
+		<AccelerateBuildsInVisualStudio>true</AccelerateBuildsInVisualStudio>
```

Comment:
Let's add a template option for this and make it enabled by default

Reply:
I'm not sure we should be enabling this by default. It's been causing more trouble than it solved, and we probably should not bother users with it at this point. Maybe we can add it commented out, so users that want to poke with it can do so.

Reply:
Given the intent from the VS team is that this will eventually become the default and that it's probably not going to result in a significant savings for an Uno project (since devs will mostly be changing the class library), I think we just close this PR for the moment. Let's keep things simple for developers